actiontext/CHANGELOG.md

```diff
@@ -1,3 +1,30 @@
+*   Capture block content for form helper methods
+
+    ```ruby
```

Comment:
Do you think the examples in the CHANGELOG and the method docs should be reworked to be erb to illustrate how the blocks will likely be used in an application?